src/tests/functional/plugin/shared/src/snippets/matmul.cpp

```diff
@@ -71,6 +71,48 @@ void MatMulBias::SetUp() {
     }
 }
 
+void MatMulBiasQuantized::SetUp() {
```

Comment:
a lot of copypaste in each SetUp override. Can it be extracted at least into MatMul::SetUp() base impl?